core/src/main/java/hudson/FilePath.java

```diff
@@ -1121,7 +1122,7 @@ public URI toURI() throws IOException, InterruptedException {
         return act(new SecureFileCallable<URI>() {
             private static final long serialVersionUID = 1L;
             public URI invoke(File f, VirtualChannel channel) {
-                return f.toURI();
+                return f.toPath().toUri();
```

Comment:
I don't see a tangible benefit of this change - can anyone explain?  It is listed as the 'new' equivalent of the old method [here](https://docs.oracle.com/javase/tutorial/essential/io/legacy.html)

Reply:
There is no strong benefit. However, in your case you are at risk of a new `InvalidPathException`, which may be thrown by `toPath()`